docs/sources/panels/transformations/types-options.md

```diff
@@ -0,0 +1,285 @@
+
+{{< docs-imagebox img="/img/docs/transformations/join-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+## Add field from calculation
+
+Use this transformation to add a new field calculated from two other fields. Each transformation allows you to add one new field.
+
+- **Mode -** Select a mode:
+  - **Reduce row -** Apply selected calculation on each row of selected fields independently.
+  - **Binary option -** Apply basic math operation(sum, multiply, etc) on values in a single row from two selected fields.
+- **Field name -** Select the names of fields you want to use in the calculation for the new field.
+- **Calculation -** Select a calculation to use when Grafana creates the new field. Click in the field to see a list of calculation choices. For information about available calculations, refer to the [Calculation list]({{< relref "../calculations-list.md" >}}).
+- **Alias -** (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
+- **Replace all fields -** (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
+
+In the example below, I added two fields together and named them Sum.
+
+{{< docs-imagebox img="/img/docs/transformations/add-field-from-calc-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+## Labels to fields
+
+> **Note:** In order to apply this transformation, your query needs to returns labeled fields.
+
+When you select this transformation, Grafana automatically transforms all labeled data into fields.
+
+Example: Given a query result of two time series
+
+1: labels Server=Server A, Datacenter=EU
+2: labels Server=Server B, Datacenter=EU
+
+This would result in a table like this
+
+| Time                | Server   | Datacenter | Value |
+| ------------------- | -------- | ---------- | ----- |
+| 2020-07-07 11:34:20 | Server A | EU         | 1     |
+| 2020-07-07 11:34:20 | Server B | EU         | 2     |
+
+**Value field name**
+If you where to select `Server` as in the **Value field name** you would get one field for every value of the `Server`
+label.
+
+| Time                | Datacenter | Server A | Server B |
+| ------------------- | ---------- | -------- | -------- |
+| 2020-07-07 11:34:20 | EU         | 1        | 2        |
+
+For this example, I manually defined labels in the Random Walk visualization of TestData DB.
+
+{{< docs-imagebox img="/img/docs/transformations/labels-to-fields-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+After I apply the transformation, my labels appear in the table as fields.
+
+{{< docs-imagebox img="/img/docs/transformations/labels-to-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+## Group By
```

Comment:
```suggestion
## Group by
```

Reply:
For this transformation, you see a list of fields and can choose to Group by or Calculate one or more columns. I don't feel like we really explain that well. The examples are good, so we know what we can do, but we don't actually tell users about what they see in the UI or how to interact with it. Care to take a crack at it?